tests/test_s3.py

```diff
@@ -776,6 +771,44 @@ class S3ConfigSource(ConfigTest):
 
     maxDiff = None
 
+    def test_normalize_initial_state(self):
+        """Check for describe/config parity after bucket creation, before changing properties"""
+
+        self.patch(s3.S3, "executor_factory", MainThreadExecutor)
+        augments = list(s3.S3_AUGMENT_TABLE)
+        augments.remove((
+            "get_bucket_location", "Location", {}, None, 's3:GetBucketLocation'))
+        self.patch(s3, "S3_AUGMENT_TABLE", augments)
+
+        bname = "custodian-test-s3confignormalize"
+        session_factory = self.replay_flight_data("test_s3_normalize_initstate", region="us-east-1")
+        session = session_factory()
+
+        queue_url = self.initialize_config_subscriber(session)
```

Comment:
it would be nice to get a terraform module for this stuff so we can avoid the extraneous record data on the initialize, but just noting as something for future enhancement in this context.